facebed.py

```diff
@@ -522,19 +522,18 @@ def process_post(post_path: str) -> ParsedPost:
         html_parser = BeautifulSoup(http_response.text, 'html.parser')
         content_node = ReelsParser.get_content_node(html_parser)
 
-        video_meta = content_node['attachments'][0]['media']
-
         video_link = ReelsParser.get_video_link(html_parser)
         video_id = content_node['id']
-        is_ig = video_meta['owner']['__typename'].startswith('InstagramUser')
-        op_name = ('📷 @' if is_ig else '') + video_meta['owner']['username' if is_ig else 'name']
-        post_url = video_meta['shareable_url']
+        owner_info = content_node['short_form_video_context']['video_owner']
```

Comment:
If `short_form_video_context` is None, this section raises an exception. It may be a bit more effective to catch the `TypeError`, so you won't have to engage in excessive defensive programming.